pl_bolts/models/rl/__init__.py

```diff
@@ -5,4 +5,4 @@
 from pl_bolts.models.rl.noisy_dqn_model import NoisyDQN
 from pl_bolts.models.rl.per_dqn_model import PERDQN
 from pl_bolts.models.rl.reinforce_model import Reinforce
-from pl_bolts.models.rl.vanilla_policy_gradient_model import PolicyGradient
+# from pl_bolts.models.rl.vanilla_policy_gradient_model import PolicyGradient
```

Comment:
why did you change this one?

Reply:
I meant to raise an issue with this, some of these imports in the __inits__ are raising errors in my runs. I meant to look into specifically why it was happening. Will update this